src/styles/blocks/about.scss

```diff
@@ -0,0 +1,60 @@
+.about {
+  display: flex;
+  flex-direction: column;
+
+  &__accent-image {
+    background-image: url(../images/player.png);
+    margin-inline: -20px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    min-height: 200px;
+
+    @media (min-width: 744px) {
+      margin-inline: -72px;
+      min-height: 400px;
+    }
+
+    @media (min-width: 1260px) {
+      margin-inline: -120px;
```

Comment:
Using minus margins is not a good idea, try another way of positioning. ( **For all files** )